src/plugins/discover/public/application/main/hooks/use_esql_mode.test.tsx

```diff
@@ -166,7 +167,54 @@ describe('useEsqlMode', () => {
       ],
       query: { esql: 'from the-data-view-2' },
     });
+    await waitFor(() => expect(replaceUrlState).toHaveBeenCalledTimes(1));
+
+    await waitFor(() => {
+      expect(replaceUrlState).toHaveBeenCalledWith({
+        columns: [],
+      });
+    });
+  });
+
+  test('changing a text based query with no transformational commands should not change state when loading and finished if index pattern is the same', async () => {
```

Comment:
```suggestion
  test('changing a ES|QL query with no transformational commands should not change state when loading and finished if index pattern is the same', async () => {
```
nit: to align with other wording in this file